apps/codeforafrica/src/payload/globals/Footer.js

```diff
@@ -0,0 +1,168 @@
+import { array } from "payload/dist/fields/validations";
+
+import linkArray from "../fields/links/linkArray";
+
+const Footer = {
+  slug: "footer",
+  access: {
+    read: () => true,
+  },
+  label: "Footer",
+  fields: [
+    {
+      name: "logo",
+      type: "upload",
+      relationTo: "media",
+      required: true,
+      filterOptions: {
+        mimeType: { contains: "image" },
+      },
+      label: "Logo",
+    },
+    {
+      name: "description",
+      type: "code",
+      label: "Description",
+      required: true,
+    },
+    {
+      type: "group",
+      name: "connect",
+      label: "Social Accounts",
+      fields: [
+        {
+          name: "title",
+          type: "text",
+          label: "Title",
+          admin: {
+            description: () =>
+              "Text that appears on contact links e.g Stay in Touch",
+          },
+
+          required: true,
+        },
+        {
+          name: "links",
+          label: "Links to social media",
+          type: "array",
+          fields: [
+            {
+              name: "media",
+              label: "Media",
+              type: "select",
+              options: [
+                {
+                  value: "facebook",
+                  label: "Facebook",
+                },
+                {
+                  value: "github",
+                  label: "Github",
+                },
+                {
+                  value: "instagram",
+                  label: "Instagram",
+                },
+                {
+                  value: "linkedin",
+                  label: "LinkedIn",
+                },
+                {
+                  value: "slack",
+                  label: "Slack",
+                },
+                {
+                  value: "twitter",
+                  label: "Twitter",
+                },
+              ],
+              unique: true,
+              required: true,
+              admin: {
+                isClearable: false,
+                isSortable: true,
+              },
+              validate: (val, options) => {
+                const { data, t } = options || {};
+                if (
+                  data?.connect?.links?.filter((l) => l.media === val)?.length >
+                  1
+                ) {
+                  return t("codeforafrica.site:uniqueMedia");
+                }
+                return array(val, options);
+              },
+            },
+            {
+              name: "url",
+              label: "URL",
+              type: "text",
+              required: true,
+              admin: {
+                description: () => "Full URL",
+              },
+            },
+            {
+              name: "icon",
+              label: "Icon",
+              type: "upload",
+              relationTo: "media",
+              required: true,
+              filterOptions: {
+                mimeType: { contains: "image" },
+              },
+            },
```

Comment:
Since we know which social media platforms we need to support, do we need to ask for icons?